recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "tiledbsoma" %}
-{% set version = "1.7.0" %}
-{% set sha256 = "7c712d03bbc511936816130f4e2af1ee6f467f2426c69f118042975a2175fdf0" %}
+{% set version = "1.7.1" %}
+{% set sha256 = "7b92a6aaff4b859f2f402d581cf895c7cf3061fe8291d7d829ebb59da1660ee8" %}
```

Comment:
I do not understand how this diff was generated. These are already the current values in `main`

https://github.com/TileDB-Inc/tiledbsoma-feedstock/blob/main/recipe/meta.yaml

https://github.com/TileDB-Inc/tiledbsoma-feedstock/blob/89a62d76950db7ecfb6909fc443407d2011f36d8/recipe/meta.yaml#L2-L3

So why does Git show that this PR is updating the lines to their current values?

Reply:
@jdblischak my apologies, manual error. Will resolve tomorrow.

Reply:
@jdblischak my apologies. I think I hadn't done `git pull` of `main` on the box I used to apply your #83 patch. I'll rebase this PR on top of current `main`.